src/configuration.rs

```diff
@@ -0,0 +1,42 @@
+use crate::model::configurable::*;
+
+/// Sets the container image to pull
+pub const CONTAINER_IMAGE: &ConfigurableItem<&str> = &ConfigurableItem {
+    environment_variable: "CONTAINER_IMAGE",
+    default_value: "ghcr.io/theconner/rustbot-runner:latest",
+};
+
+/// Sets the maximum amount of virtual CPUs available to the child container
+pub const CONTAINER_CPU: &ConfigurableItem<&str> = &ConfigurableItem {
+    environment_variable: "CONTAINER_CPU",
+    default_value: "0.5", // you get 1/2 of a cpu, i'm being generous
```

Comment:
:laughing:

Reply:
Something worth noting here: if you're running using docker-in-docker these limits won't apply (see [here](https://github.com/Summer-of-Rust/ferris-bot/pull/6/commits/26ee1ff71419155b67afa87c6a7bcae2984580c3#diff-bac1ed91dd1c520d98b8bb94164bac1240b9d076e21cfd295c50f374c8e7c7d5R33)) -- resource limitations would need to be applied to the outer container. 

(of course, if there's a workaround for this now this limitation can be resolved)